docs/install-include.rst

```diff
@@ -86,3 +89,64 @@ Select your preferences and run the install command.
                  pip install --pre --upgrade mxnet-cu100
                  git clone https://github.com/awslabs/autogluon
                  cd autogluon && python setup.py install --user
+
+     .. container:: mac
+
+        .. container:: pip
+
+           .. container:: cpu
+           
+              .. note::
+              
+                 If you don't have them, please first install: 
+                 `XCode <https://developer.apple.com/xcode/>`_, `Homebrew <https://brew.sh>`_, `LibOMP <https://formulae.brew.sh/formula/libomp>`_.
+                 Once you have Homebrew, LibOMP can be installed via:
+
+                 .. code-block:: bash
+
+                     brew install libomp
+
+              .. code-block:: bash
+
+                 pip install --upgrade mxnet
+                 pip install autogluon
+
+              .. note::
+              
+                 AutoGluon is not yet fully functional on Mac OSX. If you encounter MXNet system errors, please use Linux instead.  However, you can currently use AutoGluon for less compute-intensive TabularPrediction tasks on your Mac laptop (but only with hyperparameter_tune = False).
+
+           .. container:: gpu
+              
+              .. note::
+
+                 GPU usage is not yet supported on Mac OSX, please use Linux to utilize GPUs in AutoGluon.
+
+        .. container:: source
+
+           .. container:: cpu
+
+              .. note::
+              
+                 If you don't have them, please first install: 
+                 `XCode <https://developer.apple.com/xcode/>`_, `Homebrew <https://brew.sh>`_, `LibOMP <https://formulae.brew.sh/formula/libomp>`_.
+                 Once you have Homebrew, LibOMP can be installed via:
+
+                 .. code-block:: bash
+
+                     brew install libomp
+
+              .. code-block:: bash
+
+                 pip install --upgrade mxnet
+                 git clone https://github.com/awslabs/autogluon
+                 cd autogluon && python setup.py install --user
```

Comment:
Confirm if this actually works, likely it needs to be changed to:

cd autogluon && python setup.py develop --user

Reply:
`python setup.py install --user` is deprecated. Please use `pip install --user .`, or `pip install --user -e .` for the development install.

Reply:
> Confirm if this actually works, likely it needs to be changed to:
> 
> cd autogluon && python setup.py develop --user

For now it's just left at: python setup.py develop
which we at least know is stable.